guides/common/modules/proc_configuring-dns-dhcp-and-tftp.adoc

```diff
@@ -53,7 +53,7 @@ ifeval::["{context}" == "{project-context}"]
 --foreman-proxy-dhcp-nameservers __172.17.13.2__ \
 --foreman-proxy-tftp true \
 --foreman-proxy-tftp-managed true \
---foreman-proxy-tftp-servername $(hostname)
+--foreman-proxy-tftp-servername _{smartproxy-example-com}_
```

Comment:
I'd suggest to use IP addresses. The code to do the lookups is complex and can lead to failures. The satellite/capsule servers should have static IPs and is thus more reliable without additional maintenance overhead.

Reply:
Thank you @ekohl, I will change the option to use an IP address.